### PR TITLE
Additional row in sp_statistics_100 for heap table

### PR DESCRIPTION
**Pull request by shalinilohia50**
In Tsql, sp_statistics_100 outputs an extra row
with TYPE=0 when a table has no clustered index. This
will always  be the case in Babelfish, but sp_statistics_100
currently does not output that row.

Task: BABEL-3011
Signed-off-by: Shalini Lohia <lshalini@amazon.com>
Co-authored-by: Shalini Lohia <lshalini@amazon.com>

### CODE REVIEW

**Review by rishabhtanwar29 — APPROVED** [review]

**Review by rishabhtanwar29 — CHANGES_REQUESTED** [review]
Please merge babelfish-for-postgresql:REL2_X_DEV to jira-babel-3011 to test upgrade script changes.

**Review by Deepesh125 — DISMISSED** [review]

**Comment by shalinilohia50** [comment]
> 

Merged babelfish-for-postgresql:REL2_X_DEV to jira-babel-3011

**Review by KushaalShroff — DISMISSED** [review]

**Review by thephantomthief — DISMISSED** [review]

File: contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--2.0.0--2.1.0.sql [hunk 1]
```diff
@@ -836,5 +836,72 @@ $$
 LANGUAGE 'pltsql';
 GRANT ALL ON PROCEDURE sys.sp_sproc_columns_100 TO PUBLIC;
  
+CREATE OR REPLACE PROCEDURE sys.sp_statistics(
```
Comment (surendravishnoi) [inline contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--2.0.0--2.1.0.sql:839]: we should validate if replacing procedure can fail MVU in any way. If it is used inside a view or similar construct

Comment (shalinilohia50) [reply contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--2.0.0--2.1.0.sql:839]: This procedure is not being used in any view or similar construct, I don't think this is going to fail in MVU.

**Review by HarshLunagariya — APPROVED** [review]

**Review by xhfanhe — APPROVED** [review]

**Review by HarshLunagariya — APPROVED** [review]

**Review by kuntalghosh — APPROVED** [review]

**Pull request merged**